pkg/master/master.go

```diff
@@ -177,15 +169,24 @@ func (m *Master) InstallAPIs(c *Config) {
 			Help: "The time since the last successful synchronization of the SSH tunnels for proxy requests.",
 		}, func() float64 { return float64(m.tunneler.SecondsSinceSync()) })
 	}
+	apiserver.InstallSupport(m.MuxHelper, m.RootWebService, c.EnableProfiling, healthzChecks...)
+
+	// Install v1 unless disabled.
+	if !m.ApiGroupVersionOverrides["api/v1"].Disable {
+		// Register v1 API.
```

Comment:
I'd prefer not calling this process "register". Because in #18925, we use "register" to refer to the process that the `install` package notifies the `reigstered` package about the existence of a version. I'd suggest make this comment more specific, like "add v1 to groupVersionInfo". WDYT?